test/e2e/mv3/multiple-restarts.spec.js

```diff
@@ -318,14 +326,15 @@ describe('MV3 - Restart service worker multiple times', function () {
     }
 
     async function assertNumberOfTransactionsInPopUp(driver, number) {
+      await sleepSeconds(regularDelayMs);
```

Comment:
can we use `driver.delay` here?

Reply:
Better yet, can we just await the creation of the window with the appropriate title?